extensions/GlobalMessages/GlobalMessagesW.i18n.php

```diff
@@ -208,10 +208,10 @@
 	'wikia-language-top-list' => 'de,en,es,he,fr,it,ja,no,pl,pt,pt-br,zh',
 	'wikia-language-top' => 'Top $1 languages',
 	'wikia-language-all' => 'All languages',
-	'wikia-seconds-ago' => '$1 {{PLURAL:$1|second|seconds}} ago',
-	'wikia-minutes-ago' => '$1 {{PLURAL:$1|minute|minutes}} ago',
-	'wikia-hours-ago' => '$1 {{PLURAL:$1|hour|hours}} ago',
-	'wikia-days-ago' => '$1 {{PLURAL:$1|day|days}} ago',
+	'wikia-seconds-ago' => '$1{{PLURAL:$1|s|s}}',
```

Comment:
if the plural and singular version does not differ, do we have to use {{PLURAL}} helper?